dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -329,6 +335,78 @@ public void Test_Non_Admin_User_With_Portlet_Read_Attempt() throws DotDataExcept
         api.saveSecrets(builder.build(), APILocator.systemHost() , nonAdminUserWithAccessToPortlet);
     }
 
+    @Test
+    public void Test_BytesToChars_No_Middle_String_Conversion() throws IOException {
+
+        final AppsAPIImpl impl = new AppsAPIImpl();
+
+        testUTFCharsSet(0,127, impl);
+        testUTFCharsSet(128,255, impl);
+        testUTFCharsSet(256,383, impl);
+        testUTFCharsSet(384,591, impl);
+        testUTFCharsSet(688,767, impl);
+        testUTFCharsSet(880,1023, impl);
+        testUTFCharsSet(1024,1279, impl);
+        testUTFCharsSet(1280,1327, impl);
+        testUTFCharsSet(8192,8303, impl);
+        testUTFCharsSet(8352,8399, impl);
+        testUTFCharsSet(8448,8527, impl);
+        testUTFCharsSet(8592,8703, impl);
+        testUTFCharsSet(8704,8959, impl);
+        testUTFCharsSet(9472,9599, impl);
+        testUTFCharsSet(9600,9631, impl);
+        testUTFCharsSet(9632,9727, impl);
+        testUTFCharsSet(9728,9983, impl);
+        testUTFCharsSet(9984,10175, impl);
+    }
+
+    private void testUTFCharsSet(final int fromCode, final int toCode, final AppsAPIImpl impl) throws IOException{
+        final StringBuilder stringBuilder = new StringBuilder();
+        for(int i=fromCode; i<= toCode; i++) {
+            final String string = fromCharCode(i);
+            stringBuilder.append(string);
+        }
+        final String in = stringBuilder.toString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like in](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)